src/core/bst.h

```diff
@@ -163,8 +108,8 @@ static const char* stringify(int n) {
 class BSTVisitor;
 class ExprVisitor;
 class StmtVisitor;
-class SliceVisitor;
-class BST_keyword;
+
+static constexpr int VREG_UNDEFINED = std::numeric_limits<int>::min();
```

Comment:
Small comment here would be nice, especially if this is pulling double duty as both a source and a destination vreg.